Loop/DerivedAssets.xcassets/Contents.json

```diff
@@ -1,6 +1,6 @@
 {
   "info" : {
-    "version" : 1,
```

Comment:
This "no-op" change is just a file that keeps showing up dirty in my workspace.  Hopefully this fixes it.